pkg/limayaml/defaults.go

```diff
@@ -171,6 +172,15 @@ func defaultGuestInstallPrefix() string {
 //   - CACertificates Files and Certs are uniquely appended in d, y, o order
 func FillDefault(y, d, o *LimaYAML, filePath string) {
 	instDir := filepath.Dir(filePath)
+
+	var existingLimaVersion string // empty if the instance was created with Lima prior to v0.20
+	limaVersionFile := filepath.Join(instDir, filenames.LimaVersion)
+	if b, err := os.ReadFile(limaVersionFile); err == nil {
+		existingLimaVersion = strings.TrimSpace(string(b))
+	} else if !errors.Is(err, os.ErrNotExist) {
+		logrus.WithError(err).Warnf("Failed to read %q", limaVersionFile)
+	}
```

Comment:
Could this be factored out into a shared function in the `store` package?

Reply:
Can be done later, when the logic has to be duplicated in another place?